src/SolutionAssemblyInfo.cs

```diff
@@ -1,38 +0,0 @@
-﻿// --------------------------------------------------------------------------------------------------------------------
```

Comment:
>@AArnott said...
You mean GitVersion actually modifies source files tracked by source control rather than adding a generated file to the build?

That depends.  If you are using the GitVersionTask then no, this doesn't happen.  Instead, is generates the attributes, similar (by the sounds of it similar to how NB.GV does it).  If you are using the GitVersion.exe (which is what I normally use) then yes, you have the option of updating an existing file within Source Control.  We can have a long debate about whether or not you think that this is a good idea or not 😄 I tend to use this approach, but I only do this when I am building on a tagged master branch within my CI system.  That is really the only time that I care about the files being updated, and the generated assemblies containing the asserted version number.